vm-setup/roles/v1a2_integration_test/templates/controlplane_centos.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: {{ CLUSTER_NAME }}-controlplane-0
+  labels:
+    cluster.x-k8s.io/control-plane: "true"
+    cluster.x-k8s.io/cluster-name: "{{ CLUSTER_NAME }}"
+spec:
+  version: {{ KUBERNETES_VERSION }}
+  bootstrap:
+    configRef:
+      apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+      kind: KubeadmConfig
+      name: {{ CLUSTER_NAME }}-controlplane-0
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: BareMetalMachine
+    name: {{ CLUSTER_NAME }}-controlplane-0
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: BareMetalMachine
+metadata:
+  name: {{ CLUSTER_NAME }}-controlplane-0
+spec:
+  image:
+    url: {{ IMAGE_URL }}
+    checksum: {{ IMAGE_CHECKSUM }}
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+kind: KubeadmConfig
+metadata:
+  name: {{ CLUSTER_NAME }}-controlplane-0
+spec:
+  initConfiguration:
+    nodeRegistration:
+      name: {{ " '{{ ds.meta_data.name }}' " }}
+      kubeletExtraArgs:
+        node-labels: 'metal3.io/uuid={{ '{{ ds.meta_data.uuid }}' }}'
+  preKubeadmCommands:
+    - ifup eth1
+    - yum check-update
+    - yum install -y gcc kernel-headers kernel-devel
+    - yum update -y
```

Comment:
I would suggest to invert this line with L42 and merge the install of keepalived in the same line

Reply:
each yum install is split for readability purpose

Reply:
I'm not a fan of repeating the same command in multiple lines unless it's needed for a technical reason
If we want more readability, we can split one package (or group of packages) per line, maybe in alphabetical order or just use a list, even an external one
I suggest you put the update command first anyway

Reply:
Okay, you are right. Merging those packages